Explorer/Assets/DCL/Ipfs/EntityDefinitionGeneric.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+
+namespace DCL.Ipfs
+{
+    [Serializable]
+    public class EntityDefinitionGeneric<T> : IEquatable<EntityDefinitionGeneric<T>>
```

Comment:
Should we convert this class and its childs into structs?

Reply:
We may consider that but it's not critical at the moment as all the scene entities are persistent: they are never cleaned-up. Moreover if you do so, you won't be able to inherit from structure and thus you won't be able to serialize/deserialize with Unity's JsonUtility